src/Microsoft.OData.ModelBuilder/Microsoft.OData.ModelBuilder.xml

```diff
@@ -5106,5 +5116,2153 @@
+            <inheritdoc/>
+        </member>
+        <member name="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration">
+            <summary>
+            Org.OData.Capabilities.V1.CallbackProtocol
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration.#ctor">
+            <summary>
+            Creates a new instance of <see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration"/>
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration.HasId(System.String)">
+            <summary>
+            Protocol Identifier
+            </summary>
+            <param name="id">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration.HasUrlTemplate(System.String)">
+            <summary>
+            URL Template including parameters. Parameters are enclosed in curly braces {} as defined in RFC6570
+            </summary>
+            <param name="urlTemplate">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration.HasDocumentationUrl(System.String)">
+            <summary>
+            Human readable description of the meaning of the URL Template parameters
+            </summary>
+            <param name="documentationUrl">The value to set</param>
+            <returns><see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration"/></returns>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration.ToEdmExpression">
+            <inheritdoc/>
+        </member>
+        <member name="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackSupportedConfiguration">
+            <summary>
+            Supports callbacks for the specified protocols
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackSupportedConfiguration.#ctor">
+            <summary>
+            Creates a new instance of <see cref="T:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackSupportedConfiguration"/>
+            </summary>
+        </member>
+        <member name="M:Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackSupportedConfiguration.AddCallbackProtocols(Microsoft.OData.ModelBuilder.Capabilities.V1.CallbackProtocolConfiguration[])">
+            <summary>
+            List of supported callback protocols, e.g. `http` or `wss`
```

Comment:
@g2mula Docstring as is feels more like the docstring for the `callbackProtocols` param. A better suited docstring would be something like "Adds..."

Reply:
Unfortunately this is the documentation for the collection, I could add the ```Adds <rest of documentation here>...``` but that would break other collection documentation for example ```ExpandRestrictions``` has a property ```NonExpandableProperties``` for which the documentation is ```These properties cannot be used in expand expressions``` 
If I were to prefix that with ```Adds...``` It would also kinda not make sense.
Of course we could use some smart language analysis thingies to do this but I think it would be too much effort for little gain...
A better option would be to fix the documentation at source :-)